src/Wordpress/CustomPostType/Map.php

```diff
@@ -20,355 +18,610 @@ class Map extends CustomPostType {
+				'type' => 'text_small',
+			),
+			array(
+				'name' => esc_html__( 'address search bounds - left bottom', 'commonsbooking' ) . ' ' . esc_html__( 'latitude', 'commonsbooking' ),
+				'desc' => esc_html__( 'defines the bounds of the address search - set the bottom left corner of the bounding box', 'commonsbooking' ),
+				'id'   => 'address_search_bounds_left_bottom_lat',
+				'type' => 'text_small',
+			),
+			array(
+				'name' => esc_html__( 'address search bounds - right top', 'commonsbooking' ) . ' ' . esc_html__( 'longitude', 'commonsbooking' ),
+				'desc' => esc_html__( 'defines the bounds of the address search - set the longitude of the right top corner of the bounding box', 'commonsbooking' ),
+				'id'   => 'address_search_bounds_right_top_lon',
+				'type' => 'text_small',
+			),
+			array(
+				'name' => esc_html__( 'address search bounds - right top', 'commonsbooking' ) . ' ' . esc_html__( 'latitude', 'commonsbooking' ),
+				'desc' => esc_html__( 'defines the bounds of the address search - set the latitude of the right top corner of the bounding box', 'commonsbooking' ),
+				'id'   => 'address_search_bounds_right_top_lat',
+				'type' => 'text_small',
+			),
+			array(
+				'name' => esc_html__( 'show item availability filter', 'commonsbooking' ),
+				'desc' => esc_html__( 'activate to show the item availability filter', 'commonsbooking' ),
+				'id'   => 'show_item_availability_filter',
+				'type' => 'checkbox',
+			),
+			array(
+				'name'       => esc_html__( 'label for item availability filter', 'commonsbooking' ),
+				'desc'       => esc_html__( 'alternative label for the item availability filter', 'commonsbooking' ),
+				'id'         => 'label_item_availability_filter',
+				'type'       => 'text',
+				'attributes' => array(
+					'placeholder' => esc_html__( 'availability', 'commonsbooking' ),
+				)
+			),
+			array(
+				'name'       => esc_html__( 'label for item category filter', 'commonsbooking' ),
+				'desc'       => esc_html__( 'alternative label for the item category filter', 'commonsbooking' ),
+				'id'         => 'label_item_category_filter',
+				'type'       => 'text',
+				'attributes' => array(
+					'placeholder' => esc_html__( 'categories', 'commonsbooking' ),
+				)
+			),
+			array(
+				'name'       => esc_html__( 'custom text for filter button', 'commonsbooking' ),
+				'desc'       => esc_html__( 'the text for the button used for filtering', 'commonsbooking' ),
+				'id'         => 'custom_filterbutton_label',
+				'type'       => 'text',
+				'attributes' => array(
+					'placeholder' => esc_html__( 'filter', 'commonsbooking' ),
+				)
+			),
+			array(
+				'name'       => commonsbooking_sanitizeHTML( __( 'Filter groups', 'commonsbooking' ) ),
+				'desc'       => commonsbooking_sanitizeHTML( __( 'Filter groups can group item or location categories together to allow for filtering in the map.', 'commonsbooking' ) ),
+				'id'         => 'filtergroups',
```

Comment:
Um hier die Einstellungsmöglichkeiten der alten Verison wiederherzustellen, müsste man glaube ich `filtergroups`/`groups` mit einzelnen Items erstellen. Diese könnte man dann hinzufügen und auch sortieren. Die einzelnen Items könnten entweder mit `select`/`text` eine Angabe von einzelnen Taxonomy-Items zulassen. Die Reihenfolge innerhalb der Group würde dann die Reihenfolge angeben, wie diese im Frontend im Filter-Widget der Karte sortiert sind.

In der alten Version war es glaube ein Textfeld und die Namen mussten übereinstimmen.

Reply:
Hmm, das klingt für mich sehr umständlich. Dann müsste ich ja irgendwie iterativ die Gruppen erstellen und ich weiß auch gar nicht ob das UI mäßig geht. Also ich wäre bereit, auf die Möglichkeit der Sortierung dafür zu verzichten. Man kann ja schon die Filtergruppen sortieren, warum man die einzelnen Elemente sortieren soll erschließt sich mir nicht.

Reply:
Und wenn Elemente zusammengehören wie "Elektro" und "Muskelkraft", dann kann man die ja gruppieren.

Reply:
Ok vielleicht habe ich nicht verstanden wie die Filter Group funktioniert. Ich schaue mir das nochmal an, mein Verständnis akutell war, dass Filtergruppen die Karte vorfiltern.

Reply:
Nein, das sind die Vorfilter (unten) die Filtergruppen sind z.B. das hier: 
![image](https://github.com/user-attachments/assets/8e8ad25b-535b-49d9-b309-a66716ac6f53)

Jede von diesen Boxen ist eine Kategorie.